src/QY2Graph.cc

```diff
@@ -442,6 +439,52 @@ QY2Graph::renderGraph(graph_t* graph)
 }
 
 
+QString
+QY2Graph::unescape(const std::string& s) const
+{
+    std::string r;
```

Comment:
Just some questions: How often an escape is really used? How long the strings usually are?

We are just creating a duplicate string one-by-one character. Maybe we could do something like

```c++
if (s.find('\\') == std::string::npos)
  return QString::fromUtf8(s.c_str())
```

to optimize it for not escaped strings. But if the strings are small this optimization is probably not needed...

Reply:
In the use case I know of the strings are usually not so long. Tooltips strings might be a bit longer but usually also include newlines.

In general I think any optimization for speed here is not needed.